dataframe/arrow.go

```diff
@@ -394,3 +400,12 @@ func UnmarshalArrow(b []byte) (*Frame, error) {
 	}
 	return frame, nil
 }
+
+// ToJSONString return the FieldConfig as a json string
+func toJSONString(val interface{}) string {
+	b, err := json.Marshal(val)
+	if err != nil {
+		return "{}"
```

Comment:
Should return (string, error), and return "", err if `err != nil`, else return `string(b), nil`